File: 이수창/20230513/같은 숫자는 싫어.js
function solution(arr)
{
    var answer = [];
   
    for(let i=0 ;i<arr.length;i++)
        {   if(arr[i]==arr[i-1])
               continue;
            else
                answer.push(arr[i])
        }
    
    return answer;
}

//같은 숫자면 넘기고 다른 숫자면 답에 넣는다

File: 서예진A/20230513/같은숫자는싫어.cpp
#include <vector>
#include <iostream>
#include <algorithm>

using namespace std;

vector<int> solution(vector<int> arr) 
{
    /*중복제거함수 erase사용하여 중복원소 제거함*/
    vector<int> answer;
    
    arr.erase(unique(arr.begin(),arr.end()),arr.end());
    /*unique함수 = 배열에서 중복되지 않는 원소들 앞에서부터 채워 나감*/
    
    answer = arr;

    return answer;
}